js/utils/create-card.js

```diff
@@ -0,0 +1,89 @@
+import { createSimilarAdd } from '../data.js';
+
+const RUTYPES = {
+  'palace': 'Дворец',
+  'flat': 'Квартира',
+  'house': 'Дом',
+  'bungalow': 'Бунгало',
+  'hotel': 'Отель',
+};
+
+const mapCanvas = document.querySelector('#map-canvas');
+const templateCard = document.querySelector('#card').content.querySelector('.popup');
+const similarCards = createSimilarAdd();
+
+const translateRuType = (item) => {
+  const typeRoom = item.offer.type;
+  const ruTypeRoom = RUTYPES[typeRoom];
+  return ruTypeRoom;
+};
+
+const modifireFeatures = (item, elem) => {
+  const featureData = item.offer.features || [];
+  const featureContainer = elem.querySelector('.popup__features');
+  const featureList = featureContainer.querySelectorAll('.popup__feature');
+
+  const modifires = featureData.map((feature) => `popup__feature--${feature}`);
+
+  featureList.forEach((feature) => {
+    const modifire = feature.classList[1];
+
+    if (!modifires.includes(modifire)) {
+      feature.remove();
+    }
+  });
+};
+
+const modifirePhotos = (item, elem) => {
+  const photoData = item.offer.photos || [];
+  const photoContainer = elem.querySelector('.popup__photos');
+  const photoContainerItem = photoContainer.querySelector('.popup__photo');
+  photoContainer.textContent = '';
+  photoData.forEach((photoPath) => {
+    const photoItem = photoContainerItem.cloneNode(true);
+    photoItem.src = photoPath;
+    photoContainer.append(photoItem);
+  });
+
+  if (!photoData.length) {
+    photoContainer.remove();
+  }
+};
+
+const isValue = (value, elem) => value ? value : elem.remove();
+
+
+const createAdsCards = () => {
+  const adsCards = similarCards.map((card) => {
+    const cardElement = templateCard.cloneNode(true);
+    const cardTitle = cardElement.querySelector('.popup__title');
+    const cardAdress = cardElement.querySelector('.popup__text--address');
+    const cardPrice = cardElement.querySelector('.popup__text--price');
+    const cardType = cardElement.querySelector('.popup__type');
+    const cardCapacity = cardElement.querySelector('.popup__text--capacity');
+    const cardTime = cardElement.querySelector('.popup__text--time');
+    const cardDescription = cardElement.querySelector('.popup__description');
+    const cardAvatar = cardElement.querySelector('.popup__avatar');
+
+    cardTitle.textContent = isValue(card.offer.title, cardTitle);
+    cardAdress.textContent = isValue(card.offer.adress, cardAdress);
+    cardPrice.innerHTML = isValue(`${card.offer.price} &#8381;/ночь`, cardPrice); //undefined рублей за ночь
```

Comment:
```suggestion
    cardPrice.innerHTML = isValue(`${card.offer.price} &#8381;/ночь`, cardPrice); //undefined рублей за ночь
```
ты первым аргументом передаешь уже готовую строку`${card.offer.price} &#8381;/ночь`. Не пустая Строка это всегда true. Тебе нужно проверять именно `card.offer.price`